### PR TITLE
Improvements to JetCommandLine

### PR DESCRIPTION
**Pull request by cangencer**

### CODE REVIEW

File: hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java [hunk 1]
```diff
@@ -67,9 +68,16 @@
 @Command(
         name = "jet",
         header = "Hazelcast Jet",
```
Comment (eminn) [inline hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java]: should we also include the product version in header ?

**Comment by cangencer** [comment]
verify

**Comment by Holmistr** [comment]
verify

**Comment by Holmistr** [comment]
verify

**Comment by Holmistr** [comment]
verify

**Pull request merged**